docs/source/software/spack.md

```diff
@@ -9,63 +9,132 @@ Spack software installations are part of your research and should preferably be
+any previous versions of LAMMPS, you must uninstall them before using
+this procedure. To see if you have any previous versions of LAMMPS,
+type `spack find lammps`. If you do have a previous version, you will
+need to uninstall LAMMPS by typing `spack uninstall --dependents
+lammps`. Then, you can follow the instructions below. Note that the
+installation can take about two hours to complete. As part of the
+procedure, we recommend that you initiate a [tmux session] so that
+you can have the installation running as a separate process if you
+need to do other work on Discovery. If you decide to use tmux, make
+note of the compute node number (compute node numbers start with c or
+d with four numbers, such as c0123) to make it easier to check on the
+progress of the installation.
+
+If LAMMPS has a dependency on a specific `gcc` compiler, then do the following before starting the installation procedure. This will update the `compilers.yaml` file located in `$HOME/.spack/linux`.
+
+1. `cd $HOME/.spack/linux/`
+1. Open `compilers.yaml` and copy-paste a `compiler` entry at the end of the file. 
+1. Edit 'spec' and 'path' to indicate the version of the gcc compiler that is required for installation. 
+
+   ::::{code-block} bash
+   For example:
+        spec: gcc@=8.1.0
+    	paths:
+    	  cc: /shared/centos7/gcc/8.1.0/bin/gcc
+     	  cxx: /shared/centos7/gcc/8.1.0/bin/g++
+    	  f77: /shared/centos7/gcc/8.1.0/bin/gfortran
+      	  fc: /shared/centos7/gcc/8.1.0/bin/gfortran
+   ::::
+
+1. The `compilers.yaml` file should now have the desired `gcc` version as its latest `compiler` entry.
+1. Assuming that Spack has already been installed at a desired location. For installing gpu-supported LAMMPS, request a GPU node for 8 hours:
 
    :::{code-block} shell
-   srun --partition=gpu --nodes=1 --ntasks=14 --pty --export=All --gres=gpu:1 --mem=0 --time=08:00:00 /bin/bash
+   srun --partition=gpu --nodes=1 --ntasks=14 --pty --gres=gpu:1 --mem=16GB --time=08:00:00 /bin/bash
    :::
 
-1. (Optional) Initiate a `screen` session:
+1. Load compatible cuda, gcc, and python modules and activate Spack from the installed location.
+
+   ::::{code-block} bash
+    module load cuda/10.2 gcc/8.1.0 python/3.8.1
+    export SPACK_ROOT=/work/<PI-Project-Dir>/spack
+    . $SPACK_ROOT/share/spack/setup-env.sh
+   :::: 
 
-   1. Type `screen -S lammps-install` to create a screen session.
-   1. Type `screen -ls` to check to see if the session was created (you'll see it listed if it was successfully created).
-   1. Type `screen -rd lammps-install` to enter that screen session.
-   1. Type `echo $STY` to check that you are in the screen session.
-   1. Type `CTRL+A+D` to exit the screen.
+1. (Optional) Initiate a `tmux` session:
+
+   1. Start a tmux session: `tmux`.
+   1. List tmux sessions: `tmux ls` 
+   1. Detach from tmux session: `Ctrl+b d`
+   1. Attach to tmux session: `tmux attach-session -t 0` 
+   1. Exit a tmux session: `Ctrl+d` 
```

Comment:
Maybe change this to a bulleted list because with numbering it, it looks like you should follow them from 1 - 5.